environment.yml

```diff
@@ -24,7 +24,7 @@ dependencies:
   - iris>=2.2
   - matplotlib<3  # Can be installed from PyPI, but is a dependency of iris and should be pinned.
   - python-stratify
-  - xarray  # Can be installed from PyPI, but here to get a consistent set of depencies with iris.
+  - xarray>=0.12.0  # Can be installed from PyPI, but here to get a consistent set of depencies with iris.
```

Comment:
@bouweandela does this affect other diagnostics and/or preprocessor modules?

Reply:
Note that I put this constraint on the `xarray` package to ensure that `coarsen` function is available.

Reply:
can someone create the environment from scratch see if there's any environment solvability issues due to the restrictive versioning? Shouldn't be, but better safe than sorry